packages/shared/src/porter.ts

```diff
@@ -148,17 +150,29 @@ type PostTacoDecryptResponse = {
 };
 
 export type TacoDecryptResult = {
-  encryptedResponses: Record<string, EncryptedThresholdDecryptionResponse>;
-  errors: Record<string, string>;
+  encryptedResponses: Record<
+    ChecksumAddress,
+    EncryptedThresholdDecryptionResponse
+  >;
+  errors: Record<ChecksumAddress, string>;
 };
 
 // Signing types
+
+type PostTacoSignRequest = {
+  readonly encrypted_signing_requests: Record<
+    ChecksumAddress,
+    Base64EncodedBytes
+  >;
+  readonly threshold: number;
+};
+
 type TacoSignResponse = {
   readonly result: {
     readonly signing_results: {
```

Comment:
is this an instance of the `TacoSignResult` on line 188?

Reply:
This reminded me that `TacoSignature` should have been moved out of Porter. That is now done via https://github.com/nucypher/taco-web/pull/737/commits/aaee2226853f1c9135404250ab8491cf71412dce